src/script/interpreter.cpp

```diff
@@ -1743,7 +1759,7 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
                 case OP_INSPECTINPUTSEQUENCE:
                 case OP_INSPECTINPUTISSUANCE:
                 {
-                    // OP_INSPECTINPUT is available post tapscript
```

Comment:
Can we move all these changes, including the push4_le, back into the previous commit?

Reply:
Done. Squashed introspection-related commits into one.